README.adoc

```diff
@@ -119,6 +119,49 @@ Run unit tests:
         -v /home/my_name/my_project/my_output/heudiconv_output:/out:z \
         quay.io/rh-impact/pl-heudiconv pl-heudiconv /in /out
 
+### Example output
+
+
+[subs=+quotes]
+....
```

Comment:
I think showing the output of running againstthe following directory from upstream might be more complete, WDYT? https://github.com/nipy/heudiconv/tree/master/heudiconv/tests/data/01-fmap_acq-3mm